net-libs/libtelnet/libtelnet-9999.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-inherit cmake-utils
+inherit cmake
 
 if [[ ${PV} == *9999 ]]; then
```

Comment:
Do you want to sync the 9999 with the 0.23 ebuild? That's why this if logic exists.

Reply:
I know :) Upstream switched from autotools to cmake after 0.23, but I left the if logic for future releases

Reply:
I've got ya.